tox.ini

```diff
@@ -53,6 +51,7 @@ commands =
     check-manifest
 
 [testenv:docs]
+extras = docs
 changedir = docs
 commands =
     sphinx-build -b html . html
```

Comment:
My suggestion is fail on warnings. In this way the docs should be "clean".

```suggestion
    sphinx-build -W --keep-going  -b html . html
```

Reply:
I 100% agree, but I've never been able to figure out how to clear the remaining warning. It has been a while since I looked at it, but IIRC it is implying a cross-reference from an ivar (which makes no sense, but I've seen similar issues on other projects).